playwright-tests/util/sandboxrpc.js

```diff
@@ -463,7 +463,7 @@ export class SandboxRPC {
         args: Buffer.from(JSON.stringify(createDaoConfig)).toString("base64"),
       },
       gas: 300000000000000,
-      attachedDeposit: utils.format.parseNearAmount("8"),
+      attachedDeposit: utils.format.parseNearAmount("15"),
```

Comment:
Is this amount a new requirement for creating an instance now? Do we need to change it in the treasury factory UI too?

Reply:
Oh no no, I added a lockup test, and creating a lockup needs minimum 4N, so I had to increase the deposit while creating a DAO, to make it work